frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/object-service-dashboards.spec.ts

```diff
@@ -0,0 +1,160 @@
+const testName = `test-${Math.random()
+  .toString(36)
+  .replace(/[^a-z]+/g, '')
+  .substr(0, 5)}`;
+
+const testBucket = {
+  apiVersion: 'objectbucket.io/v1alpha1',
+  kind: 'ObjectBucketClaim',
+  metadata: {
+    namespace: 'openshift-storage',
+    name: testName,
+  },
+  spec: {
+    ssl: false,
+    storageClassName: 'openshift-storage.noobaa.io',
+    generateBucketName: 'test-bucket',
+  },
+};
+
+const installStorageCluster = () => {
+  cy.login();
+  cy.visit('/');
+  cy.install();
+};
+
+describe('Tests Buckets Card', () => {
+  before(() => {
+    installStorageCluster();
+  });
+
+  beforeEach(() => {
+    cy.visit('/');
+    cy.byLegacyTestID('horizontal-link-Object Service').click();
+  });
+  it('Test at least one noobaa bucket is present', () => {
+    cy.get('div:nth-child(1) > div.co-inventory-card__item-title')
+      .invoke('text')
+      .then((text) => {
+        const count = parseInt(text.split(' ')[0], 10);
+        expect(count).toBeGreaterThanOrEqual(1);
+      });
+  });
+
+  it('Create an Object Bucket Claim and test equality', () => {
+    cy.get('div:nth-child(3) > div.co-inventory-card__item-title')
+      .invoke('text')
+      .then((text) => {
+        const initalCount = parseInt(text.split(' ')[0], 10);
+        cy.exec(`echo '${JSON.stringify(testBucket)}' | kubectl create -f -`);
+        cy.get('div:nth-child(3) > div.co-inventory-card__item-title')
+          .invoke('text')
+          .then((data) => {
+            const finalCount = parseInt(data.split(' ')[0], 10);
+            expect(finalCount).toEqual(initalCount + 1);
+            cy.exec(`echo '${JSON.stringify(testBucket)}' | kubectl delete -f -`);
+          });
+      });
+  });
+});
+
+describe('Test Status Card', () => {
+  const isRGWPresent = () => {
+    const RGW_PROVISIONER = 'openshift-storage.ceph.rook.io/bucket';
+    cy.exec('oc get StorageClasses -o json').then((result) => {
+      if (result.code === 0) {
+        const storageClasses = JSON.parse(result.stdout.toString());
+        return storageClasses.items.some((sc) => sc.provisioner === RGW_PROVISIONER);
+      }
+      return false;
+    });
+    return false;
+  };
```

Comment:
Move it to utils.

Reply:
We are always running this test on cloud at this current point so we don't really need this. We can introduce this once we need it.